external/llvm-project/mlir/lib/Dialect/Tosa/Transforms/TosaPartition.cpp

```diff
@@ -97,10 +97,10 @@ bool isElementwiseOp(Operation *op) {
         tosa::NegateOp,
         tosa::ReciprocalOp,
         tosa::RsqrtOp,
-        tosa::SelectOp
-//         tosa::EqualOp,
-//         tosa::GreaterOp,
-//         tosa::GreaterEqualOp
+        tosa::SelectOp,
+        tosa::EqualOp,
+        tosa::GreaterOp,
+        tosa::GreaterEqualOp
```

Comment:
@pcf000 any reason these were commented out?

Reply:
Because they're covered by the ResultsBroadcastableShape trait which is tested further up.